info2439/info2439/module4/assignment4.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Camden Grove | INFO 2439 | Assignment3</title>
+  <meta charset="utf-8"> 
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+ 
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+	
+	
+	  <link rel="stylesheet" href="bob/themes/assignment4.css" />
+  <link rel="stylesheet" href="bob/themes/assignment4.min.css" />
+  
+  
+  <script src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="https://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    
+</head>
+<body>
+<!-- Camden Grove
+     Assignment3.html
+     INFO2439
+     Thoendel
+     9/29/19
+-->
+
+
+<div data-role="page">
+
+	<div data-role="header">
+	<a data-ajax="false" href="http://student.mccinfo.net/~cjgrove/info2439/index.html" class="ui-btn-left" target="_blank" data-icon="back">Back</a>
+		<h1>Assignment#4 </h1>
+	</div>
+	
+	<div role="main" class="ui-content">
+		
+		<form action="http://lolberding.portfolios.mccinfo.net/info_survey/bistrosurvey_insert.php" method="post"> 
```

Comment:
This is the wrong form action. It looks like you were following an old pdf from a previous version of the class. All my assignments are in Google Docs. The embedded instructions document (and now the linked doc) have the URL of the correct php script. You'll want to get this corrected for the challenge - as I grade you by being able to verify a write to the db. Let me know if you have questions.